### ISSUE TITLE
javadoc warning on build

### ISSUE THREAD

**Issue by scompo** [scompo/utils#8]
take a look at: https://jitpack.io/com/github/scompo/utils/1.0.4/build.log

```
[WARNING] Javadoc Warnings
[WARNING] Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF-8
[WARNING] /home/jitpack/build/src/main/java/com/github/scompo/utils/assertions/NullChecker.java:19: warning: no @param for <T>
[WARNING] public static <T> boolean isNull(T obj) {
[WARNING] ^
[WARNING] /home/jitpack/build/src/main/java/com/github/scompo/utils/assertions/NullChecker.java:34: warning: no @param for <T>
[WARNING] public static <T> boolean isNotNull(T obj) {
[WARNING] ^
```

**Comment by scompo** [comment]
Should be sufficient to add the @param annotation for the generics methods

**Comment by scompo** [comment]
fixed here 40213c977592dcb550d2936f5271f029c9f0a51a
